crates/env_types/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "ink_env_types"
```

Comment:
What is the purpose of the `ink_env_types` crate?

Reply:
See the description of this PR:
>I extracted the env types into a separate crate ink_env_types. This was necessary in order to prevent circular dependencies of ink_env depending on engine (to integrate the engine) depending on ink_env again (to use the types).